rasa_nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -22,10 +22,15 @@
 class CountVectorsFeaturizer(Featurizer):
     """Bag of words featurizer
 
-    Creates bag-of-words representation of intent features
-    using sklearn's `CountVectorizer`.
-    All tokens which consist only of digits (e.g. 123 and 99
-    but not ab12d) will be represented by a single feature."""
+        Creates bag-of-words representation of intent features
```

Comment:
please double check what the google python doc comments style guide says about this, but I think there should not be any intendation here (i.e. keep it the way it was).